### PR TITLE
Feature/opcon 22798-dk-addon

### PR DESCRIPTION
**Pull request by dan-kiss-dev-this**
Additional requested adjustments from ticket OPCON-22798

### CODE REVIEW

**Comment by hblake316** [comment]
Looks like we already have a page for managing new service requests:
https://help.smatechnologies.com/opcon/core/Files/UI/Solution-Manager/Creating-Service-Requests

File: docs/Files/UI/Solution-Manager/Library/Calendars/Calendars.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+lang: en-us
+title: Managing Calendars
+viewport: width=device-width, initial-scale=1.0
+---
+
```
Comment (hblake316) [inline docs/Files/UI/Solution-Manager/Library/Calendars/Calendars.md]: this file should be called Managing-Calendars.md, or the title needs to be Calendars

File: docs/Files/UI/Solution-Manager/Library/Departments/Departments.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (hblake316) [inline docs/Files/UI/Solution-Manager/Library/Departments/Departments.md]: this file should be named Managing-Departments.md, or the title needs to be Departments

File: docs/Files/UI/Solution-Manager/Managing-New-Service-Request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (hblake316) [inline docs/Files/UI/Solution-Manager/Managing-New-Service-Request.md]: we already have a file for new service requests: Files/UI/Solution-Manager/Creating-Service-Requests

**Review by rweesner — CHANGES_REQUESTED** [review]
Please review comments for changes

**Comment by dan-kiss-dev-this** [comment]
> Looks like we already have a page for managing new service requests: https://help.smatechnologies.com/opcon/core/Files/UI/Solution-Manager/Creating-Service-Requests

DK - Made change to use Files/UI/Solution-Manager/Creating-Service-Requests/

File: docs/Files/UI/Solution-Manager/Library/Calendars/Calendars.md (hunk 1 above)
Comment (dan-kiss-dev-this) [reply docs/Files/UI/Solution-Manager/Library/Calendars/Calendars.md]: Changed to use Managing-Calendars.md

File: docs/Files/UI/Solution-Manager/Library/Departments/Departments.md (hunk 2 above)
Comment (dan-kiss-dev-this) [reply docs/Files/UI/Solution-Manager/Library/Departments/Departments.md]: Adjusted file name to be Managing-Departments.md

File: docs/Files/UI/Solution-Manager/Managing-New-Service-Request.md (hunk 3 above)
Comment (dan-kiss-dev-this) [reply docs/Files/UI/Solution-Manager/Managing-New-Service-Request.md]: Adjusted to use Creating-Service-Requests

**Review by rweesner — APPROVED** [review]

**Pull request merged**